README.md

```diff
@@ -30,76 +41,92 @@ $ helm install ohmyglb ohmyglb/ohmyglb
 See [values.yaml](https://github.com/AbsaOSS/ohmyglb/blob/master/chart/ohmyglb/values.yaml)
 for customization options.
 
-### Local Playground
-
-####  Deploy local cluster
-
-```sh
-$ make deploy-local-cluster
+### Local Playground Install
+
+#### Environment prerequisites
+
+ - [install **GO 1.14**](https://golang.org/dl/)
+ 
+ - [install **GIT**](https://git-scm.com/downloads)     
+    
+ - install **gnu-sed** if you don't have it  
+    - If you are on a Mac, install sed by Homebrew
+    ```shell script
+    brew install gnu-sed 
+    ``` 
+   
+ - [install **Docker**](https://docs.docker.com/get-docker/)
+    - ensure you are able to push/pull from your docker registry
+    - to run multiple clusters reserve 8GB of memory  
+    
+      ![](docs/images/docker_settings.png)
+      <div>
+        <sup>above screenshot is for <strong>Docker for Mac</strong> and that options for other Docker distributions may vary</sup>
+      </div>
+
+ - [install **Kubectl**](https://kubernetes.io/docs/tasks/tools/install-kubectl/) to operate clusters
+     
+ - [install **Helm3**](https://helm.sh/docs/intro/install/) to get charts
+    - add default repo 
+    ```shell script
+    helm repo add stable https://kubernetes-charts.storage.googleapis.com
```

Comment:
we actually can move it to https://github.com/ytsarev/ohmyglb/blob/master/Makefile#L87 and omit this part of the doc completely

It's only for local nginx-ingress because other dependencies have  direct repo links in https://github.com/ytsarev/ohmyglb/blob/master/chart/ohmyglb/Chart.yaml#L25